app/controllers/users_controller.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class UsersController < WithUserController
+  before_action :get_user, only: [:update]
+
```

Comment:
this before_action is unnecessary, `before_action :authorize!` defined in `with_user_controller` will correctly populate the `@user` model for you

Reply:
done